src/core/thread/mle.hpp

```diff
@@ -1984,9 +1998,12 @@ class Mle : public InstanceLocator, private NonCopyable
                         LinkQuality            aLinkQuality,
                         uint8_t                aLinkMargin,
                         const ConnectivityTlv &aConnectivityTlv,
-                        uint8_t                aVersion,
-                        uint8_t                aCslClockAccuracy,
-                        uint8_t                aCslUncertainty);
+                        uint8_t                aVersion
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+                        ,
+                        const Mac::CslAccuracy &aCslAccuracy
+#endif
```

Comment:
How about always including this argument? I hope the compiler will optimize out unused args?

Reply:
Sounds good to me. 
We need to remove the `#if` check for `CslAccuracy` class definition, but I think it is pretty harmless.